Library/Homebrew/formula.rb

```diff
@@ -1820,10 +1820,12 @@ def std_cargo_args(root: prefix, path: ".")
       install_prefix: T.any(String, Pathname),
       install_libdir: T.any(String, Pathname),
       find_framework: String,
+      program_path:   T.nilable(T::Array[String]),
     ).returns(T::Array[String])
   }
-  def std_cmake_args(install_prefix: prefix, install_libdir: "lib", find_framework: "LAST")
-    %W[
+  def std_cmake_args(install_prefix: prefix, install_libdir: "lib", find_framework: "LAST",
+                     program_path: ENV["PATH"]&.split(File::PATH_SEPARATOR))
```

Comment:
```suggestion
  def std_cmake_args(install_prefix: prefix, install_libdir: "lib", find_framework: "LAST")
```
I suggest just using the contents of `ENV["PATH"]` unconditionally unless we know there are formulae that need to set this.